spt/features/restart.cpp

```diff
@@ -17,22 +16,28 @@ CON_COMMAND(tas_restart_game, "Restarts the game")
 	}
 }
 
-bool RestartFeature::ShouldLoadFeature()
+// Does game restarts
+class RestartFeature : public FeatureWrapper<RestartFeature>
 {
-#ifdef SPT_TAS_RESTART_ENABLED
-	return true;
-#else
-	return false;
+protected:
+	virtual bool ShouldLoadFeature()
+	{
+#if defined(OE) || defined(SSDK2013)
```

Comment:
What's the reasoning behind changing this check from ifdef SPT_TAS_RESTART_ENABLED ?

Reply:
idk how this is related to the header. Also idk if you are aware but include guard is a specific term used for definitions that prevent including the same header once, ie.
```
#ifndef XD
#define XD


// some header stuff

#endif```

Reply:
yeah I realized that include guard isn't really the right word for it. I removed restart.hpp because that's where I previously defined `SPT_TAS_RESTART_ENABLED`. But I only used that in the mesh renderer and I don't need to anymore, so `SPT_TAS_RESTART_ENABLED` isn't needed for the rest of spt.

Reply:
gotcha